lib/game/entities/Tile.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+
+import 'package:flutter/material.dart';
+import 'package:leap/leap.dart';
+
+import '../status/IgnoreGravity.dart';
```

Comment:
Let's try to use more absolute import paths instead of relative ones that is currently the more recommended approach